content/docs/command-reference/import.md

```diff
@@ -228,3 +228,35 @@ outs:
 
 See a full explanation in our [Data Registries](/doc/use-cases/data-registries)
 use case.
+
+## Example: Importing from any Git repository
+
+You can even import files from plain Git repos that are not <abbr>DVC
+repositories</abbr>. For example, let's import a dataset from
+[GSA's data repo](https://github.com/GSA/data).
+
+> Note that Git-tracked files can be imported from DVC repos as well.
+
+```dvc
+$ dvc import git@github.com:GSA/data \
+           enterprise-architecture/it-standards.csv
+Importing ...
+```
```

Comment:
````suggestion
[GSA's data repo](https://github.com/GSA/data):

```dvc
$ dvc import git@github.com:GSA/data \
           enterprise-architecture/it-standards.csv
Importing ...
```

> Note that Git-tracked files can be imported from DVC repos as well.
````

Reply:
I'll commit this but please take note in the order above for future ref. Thanks!